tests/submission/test_admin_endpoints.py

```diff
@@ -0,0 +1,148 @@
+"""
+Tests for admin endpoint functionality.
+"""
+# pylint: disable=unused-argument, no-member
+
+import json
+
+from gdcdatamodel import models as md
+from tests.submission.test_endpoints import post_example_entities_together
+from tests.submission.utils import data_fnames
+
+def create_blgsp_url(path):
+    base_url = '/v0/submission/admin/CGCI/BLGSP'
+    if not path.startswith('/'):
+        path = '/' + path
+    return base_url + path
+
+def post_blgsp_files(client, headers):
+    """Boilerplate setup code for some tests
+
+    Args:
+        client (fixture): fixture for making http requests
+        headers (dict): http header with token
+    Returns:
+        dict: map of entity type to UUID of submitted entities
+    """
+
+    test_fnames = (
+        data_fnames
+        + ['read_group.json', 'submitted_unaligned_reads.json']
+    )
+
+    entity_types = [fname.replace('.json', '') for fname in data_fnames]
+    resp = post_example_entities_together(client,
+                                          headers,
+                                          data_fnames2=test_fnames)
+    assert resp.status_code == 201, resp.data
+
+    submitted_entities = {
+        entity['type']: entity['id']
+        for entity in resp.json['entities']
+    }
+
+    for entity_type in entity_types:
+        assert entity_type in submitted_entities, 'entity not found in submission'
+
+    return submitted_entities
+
+def test_to_delete_with_admin(client, pg_driver, cgci_blgsp, submitter, admin):
+    """Try to set the sysan of a node with admin credentials
+
+    Url:
+        DELETE: /admin/<program>/<project>/entities/<ids>/to_delete/<to_delete>
+    """
+
+    entities = post_blgsp_files(client, submitter)
+    did = entities['submitted_unaligned_reads']
+
+    base_delete_path = create_blgsp_url('/entities/{}/to_delete/'.format(did))
+    to_delete_true = base_delete_path + 'true'
+    to_delete_false = base_delete_path + 'false'
+
+    resp = client.delete(to_delete_true, headers=admin)
+    assert resp.status_code == 200, resp.data
+    with pg_driver.session_scope():
+        sur_node = pg_driver.nodes(md.SubmittedUnalignedReads).first()
+        assert sur_node
+        assert sur_node.sysan.get('to_delete') is True
+
+    resp = client.delete(to_delete_false, headers=admin)
+    assert resp.status_code == 200, resp.data
+    with pg_driver.session_scope():
+        sur_node = pg_driver.nodes(md.SubmittedUnalignedReads).first()
+        assert sur_node
+        assert sur_node.sysan.get('to_delete') is False
+
+def test_to_delete_without_admin(client, pg_driver, cgci_blgsp, submitter):
```

Comment:
This is a copypaste of `test_to_delete_with_admin`, would be better to use `@pytest.mark.parametrize('creds,resp_code', [(admin, 200), (submitter, 403)])`

Reply:
It turns out you cannot use pytest fixtures in parameterized tests.

Reply:
You can parametrize over fixture names and then use `request.getfixturevalue(fixture_name)` to get the object needed

Reply:
`request` is a pytest fixture and we have established you cannot use fixtures in that way.

Reply:
other way
```@pytest.mark.parametrize('user,resp_code', [('admin', 200), ('submitter', 403)])
def test_blah(request, user, resp_code):
    user_obj = request.getfixturevalue(user)
    ....```